src/core/toga/window.py

```diff
@@ -158,6 +158,7 @@ def position(self):
         Returns:
             A ``tuple`` of (``int``, ``int``) int the from (x, y).
         """
+        self._position = self._impl.get_position()
```

Comment:
There's no need to cache `self._position` if the backend is responsible for providing a literal value.

Reply:
yes, I think it is. What would you then return from the platforms where we do not yet have an implementation? In the current code base, self._position **does** seem to have a value and it currently **is** this value that is returned by the position getter.

Reply:
If the backend doesn't have an implementation, it's *not implemented*.  We shouldn't be baking in features to the stable frontend interface to compensate for an unstable backend.

Reply:
OK, removed self._position variable